commit afc4535dcc188dc54363e28fce7e27b17ae32869
Author:     Yuan-Ting Hsieh (謝沅廷) <yuantinghsieh@gmail.com>
AuthorDate: Tue Sep 13 13:01:34 2022 -0700
Commit:     GitHub <noreply@github.com>
CommitDate: Tue Sep 13 13:01:34 2022 -0700

    Make task_request_interval and task_check_period configurable (#878)
    
    * Make task_request_interval and task_check_period configurable
    
    * Fix comments
    
    * Make default_task_fetch_interval configurable
    
    * Fix format
---
 nvflare/private/fed/client/scheduler_cmds.py | 1 +
 nvflare/private/fed/server/server_engine.py  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/nvflare/private/fed/client/scheduler_cmds.py b/nvflare/private/fed/client/scheduler_cmds.py
index b808f82e..21cd3036 100644
--- a/nvflare/private/fed/client/scheduler_cmds.py
+++ b/nvflare/private/fed/client/scheduler_cmds.py
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 import json
 from typing import List
 
diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index e147eecb..cecc8b31 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -620,6 +620,7 @@ class ServerEngine(ServerEngineInternalSpec):
         return command_conn
 
     def persist_components(self, fl_ctx: FLContext, completed: bool):
+        self.logger.info("Start saving snapshot on server.")
 
         # Call the State Persistor to persist all the component states
         # 1. call every component to generate the component states data
